weasyl/controllers/general.py

```diff
@@ -23,28 +23,33 @@ def index_(request):
 def search_(request):
     rating = define.get_rating(request.userid)
 
-    form = request.web_input(q="", find="", within="", rated=[], cat="", subcat="", backid="", nextid="")
+    q = request.params.get('q', None)
```

Comment:
`None` is the default default.

```suggestion
    q = request.params.get('q')
```

(applies to other instances too)